mrchecker-docs/documentation/Who-Is-MrChecker/Test-Framework-Modules/Core-Test-Module-Test-reports-with-logs-and-or-screenshots.asciidoc

```diff
@@ -1,3 +1,26 @@
+:toc: macro
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+toc::[]
+:idprefix:
+:idseparator: -
+:reproducible:
+:source-highlighter: rouge
+:listing-caption: Listing
+
+= Overview
+
+With MrChecker you'll run or write a lot of tests and you'll need some way to read the results. Yes, you can use the default console output, but it's not a good solution, especially when you want something more readable.
+
+In this module you'll learn about Allure logger - how to find saved logs, how to use it and different types of logger. We'll also show you Allure reports, its pros, how to set it up and use it in your project.
+
 = Allure Logger -> BFLogger
```

Comment:
Please update this file and remove all "Prior to mrchecker-core-module version 5.6.2.1:" parts